### ISSUE TITLE
Failed to provision C1 server with scaleway_instance_server

### ISSUE THREAD

**Issue by ulm0** [terraform-providers/terraform-provider-scaleway#410]
### Terraform Version

v0.12.21

### Affected Resource(s)
- scaleway_instance_server

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
provider "scaleway" {}

resource "tls_private_key" "runner_key" {
  algorithm   = "RSA"
  ecdsa_curve = "4096"
}

data "scaleway_instance_image" "docker" {
  architecture = "arm"
  name         = "docker"
  latest       = true
}

resource "scaleway_account_ssh_key" "runner_key" {
  name       = "runner_key"
  public_key = tls_private_key.runner_key.public_key_openssh
}

resource "scaleway_instance_server" "runner" {
  name              = "runner"
  image             = data.scaleway_instance_image.docker.id
  enable_dynamic_ip = true
  type              = "C1"
}

```

### Debug Output
https://gist.github.com/ulm0/7de8d8eb853d4566d9a02f5d90f704be

### Expected Behavior
Provision C1 server

### Actual Behavior
```sh
$ terraform apply -auto-approve
tls_private_key.runner_key: Refreshing state... [id=0c1bcbbac34cb4c0613ec9ec4230685f4c69c04b]
scaleway_account_ssh_key.runner_key: Refreshing state... [id=093dd0fa-299b-4bea-bab7-530b936a91b7]
data.scaleway_instance_image.docker: Refreshing state...
scaleway_account_ssh_key.runner_key: Destroying... [id=093dd0fa-299b-4bea-bab7-530b936a91b7]
scaleway_instance_server.runner: Creating...
scaleway_account_ssh_key.runner_key: Destruction complete after 1s
tls_private_key.runner_key: Destroying... [id=0c1bcbbac34cb4c0613ec9ec4230685f4c69c04b]
tls_private_key.runner_key: Destruction complete after 0s
tls_private_key.runner_key: Creating...
tls_private_key.runner_key: Creation complete after 1s [id=f00ffd49b52611245a876dcb30340c29d61f1e70]
scaleway_account_ssh_key.runner_key: Creating...
scaleway_account_ssh_key.runner_key: Creation complete after 2s [id=7f6d69f8-dc61-45fd-8708-359501c13ff4]

Error: scaleway-sdk-go: http error 400 Bad Request: Incompatible boot method 'local' for hardware 'C1'

  on main.tf line 19, in resource "scaleway_instance_server" "runner":
  19: resource "scaleway_instance_server" "runner" {
```

### Steps to Reproduce
1. `terraform apply`

**Comment by kindermoumoute** [comment]
This is fixed with #401

**Comment by kindermoumoute** [comment]
Next release should land very soon
